.github/workflows/release-drafter.yml

```
@@ -0,0 +1,22 @@
+name: Release Drafter
```

yamllint

[error] wrong new line character: expected \n

(new-lines)